shared/components/CommunityAbout/index.scss

```diff
@@ -21,16 +15,19 @@
       margin-bottom: 0px;
     }
 
-    @media (min-width: 768px) {
+    @media (min-width: $tablet-portrait-width) {
```

Comment:
confirmed over slack - andrew meant examples of media queries being in confusing places.

I'm still gonna get rid of 768px media queries!